Sources/BrowserServicesKit/PrivacyConfig/PrivacyConfigurationData.swift

```diff
@@ -120,6 +120,7 @@ public struct PrivacyConfigurationData {
         public typealias FeatureSettings = [String: Any]
         public typealias Features = [String: Feature]
         public typealias FeatureSupportedVersion = String
+        public typealias SubfeatureSettings = String
```

Comment:
Probably not for this PR, but I think we should reconsider naming. Seeing how we operate with `PrivacyFeature` , `PrivacyFeature.Feature` , `FeatureSettings` and `SubfeatureSettings` something doesn't feel that obvious to me.

Reply:
I think we should rather rename `PrivacyFeature.Feature` to `PrivacyFeature.Subfeature` but that's not the decision I want to make here.